index.js

```diff
@@ -19,9 +22,19 @@ dotenv.config(); // Set up environment variables
 const app = express();
 const port = process.env.PORT || 3000;
 
+/* eslint-disable no-new */
```

Comment:
could we add some comments here mentioning what Airlock is? The rest of the functions are loosley explained with the `console.log` statements, might be worth making a note what this is